### PR TITLE
Show bathymetry plot in notebook + some other fixes

### PR DESCRIPTION
**Pull request by navidcy**
@aekiss this PR adds the missing thumbnail.

It's not done in an automatic way, I just evaluated that particular cell.

Ideally, we need to resolve https://github.com/COSIMA/regional-mom6/issues/119

Also some fixes in docstrings (eg `rectangular_grid`) and in the explanations in the demo notebooks.

### CODE REVIEW

**Comment by navidcy** [comment]
Hm... OK, it's a failed attempt...

**Comment by navidcy** [comment]
I think I fixed it! Could somebody review this?

@aekiss, here's [a preview of the docs](https://regional-mom6--156.org.readthedocs.build/en/156/) showing that the thumbnail is there!

File: regional_mom6/regional_mom6.py [hunk 1]
```diff
@@ -1949,7 +1950,7 @@ def rectangular_brushcut(self):
             },
         }
 
-        ### Generate our dz variable. This needs to be in layer thicknesses
+        ### Generate the dz variable; need needs to be in layer thicknesses
```
Comment (angus-g) [inline regional_mom6/regional_mom6.py:1953]:
```suggestion
        ### Generate the dz variable; needs to be in layer thicknesses
```

**Review by angus-g — APPROVED** [review]
PR description doesn't capture the fix to `rectangular_hgrid`. One small typo in your doc changes, otherwise I think they're fine.

I guess this just makes the thumbnails for both notebooks the same. I suppose there's no issue with that (it's better than nothing!)

**Comment by navidcy** [comment]
With #59 one of the examples will go away anyway

**Comment by navidcy** [comment]
> PR description doesn't capture the fix to `rectangular_hgrid`.

That's a change only in the comments -- I'm adding a remark

**Comment by angus-g** [comment]
> > PR description doesn't capture the fix to `rectangular_hgrid`.
> 
> 
> 
> That's a change only in the comments -- I'm adding a remark

There was also an addition of deg2rad, no?

**Comment by navidcy** [comment]
In a comment tho, not in source code.

File: regional_mom6/regional_mom6.py [hunk 2]
```diff
@@ -296,7 +296,7 @@ def rectangular_hgrid(λ, φ):
         np.diff(λ), dλ * np.ones(np.size(λ) - 1)
     ), "provided array of longitudes must be uniformly spaced"
 
-    # dx = R * cos(φ) * np.deg2rad(dλ) / 2
+    # dx = R * cos(np.deg2rad(φ)) * np.deg2rad(dλ) / 2
```
Comment (navidcy) [inline regional_mom6/regional_mom6.py:299]: @angus-g you are referring to this `deg2rad`, right?

**Pull request merged**